docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,609 @@
+
+You can pull the data from Terraform since you exposed those via the `output.tf`.
+
+1. From the Terraform code location `test/resources/aws/2-region/terraform`, execute the following to export the access keys to environment variables. This will allow an easier creation of the Kubernetes secret via the command line:
+
+```bash
+export ACCESS_KEY=$(terraform output -raw s3_aws_access_key)
+export SECRET_ACCESS_KEY=$(terraform output -raw s3_aws_secret_access_key)
+```
+
+2. From the root of the repository execute the script [create_elasticsearch_secrets.sh](https://github.com/camunda/c8-multi-region/blob/main/create_elasticsearch_secrets.sh). It will use the exported environment variables from step 1 to create the required secret within the Camunda namespaces. Those have previously been defined and exported via the [environment prerequisites](#environment-prerequisites).
+
+```bash
+./create_elasticsearch_secrets.sh
+```
+
+3. Unset environment variables to reduce the risk of potential exposure. The script is spawned in a subshell and can't modify the environment variables without extra workarounds.
+
+```bash
+unset ACCESS_KEY
+unset SECRET_ACCESS_KEY
+```
+
+### Camunda 8 Helm chart prerequisites
+
+Within the cloned repository, navigate to `test/resources/aws/2-region/kubernetes`. This contains a dual-region example setup.
+
+#### Content Elaboration
+
+Our approach is to work with layered helm values files:
+
+- have a base `camunda-values.yml` that is generally applicable for both Camunda installations
+- two overlays that are for region 0 and region 1 installations
+
+##### camunda-values.yml
+
+This forms the base layer that contains the basic required setup, which applies to both regions.
+
+Key changes of the dual-region setup:
+
+- `global.multiregion.regions: 2`
+  - indicates the use for two regions
+- `global.identity.auth.enabled: false`
+  - Identity is currently not supported. Please see the [limitations section](#) on the dual-region concept page. <!-- TODO: fill link -->.
+- `global.elasticsearch.disableExporter: true`
+  - disables the automatic Elasticsearch configuration of the helm chart. We will manually supply the values via environment variables.
+- `identity.enabled: false`
+  - Identity is currently not supported.
+- `optimize.enabled: false`
+  - Optimize is currently not supported. It has a dependency on identity.
+- `zeebe.env`
+  - `ZEEBE_BROKER_CLUSTER_INITIALCONTACTPOINTS`
+    - These are the contact points for the brokers to know how to form the cluster. Find more information on what the variable means in [setting up a cluster](../../../../zeebe-deployment/operations/setting-up-a-cluster.md).
+  - `ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION0_ARGS_URL`
+    - The Elasticsearch endpoint for region 0.
+  - `ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION1_ARGS_URL`
+    - The Elasticsearch endpoint for region 1.
+- Following cluster of 8 is recommend for a dual-region setup
+  - `zeebe.clusterSize: 8`
```

Comment:
did you double check with Consulting/Zeebe that 8/8/8 is the correct setup? I had a vague memory of the hack week that not all of these values are 8, but I could be wrong.

Reply:
you're right, sorry.
it's 8/8/4, my `zbctl status` shows the correct one.